app/inpututils.cpp

```diff
@@ -448,6 +458,12 @@ QString InputUtils::renameWithDateTime( const QString &srcPath, const QDateTime
   return QString();
 }
 
+QString InputUtils::downloadInProgressFilePath( const QString &projectDir )
+{
+  // Use first 11 characters of sha-256 of text "Input" to create unique file name
+  return projectDir + "/.mergin/.project-36ecb4f8669.downloading";
```

Comment:
not sure if I understand why the hash should be in filename at all? why not something like `.project.downloading`

Reply:
Well, I just wanted the file to be unique as much as possible :grin: , so that geopackages that are in this folder would not have the same name.

Reply:
as Peter suggests I would also go with a simpler naming :-)